user-interface/src/data-verification/transfer/PendingTransferOrder.tsx

```diff
@@ -0,0 +1,200 @@
+import { useRef, useState } from 'react';
+import { CaseSummary } from '@common/cams/cases';
+import { OrderStatus, TransferOrder, TransferOrderAction } from '@common/cams/orders';
+import { AlertDetails, UswdsAlertStyle } from '@/lib/components/uswds/Alert';
+import { useApi } from '@/lib/hooks/UseApi';
+import { TransferConfirmationModal } from './TransferConfirmationModal';
+import { ConfirmationModalImperative } from '../ConsolidationOrderModal';
+import Button, { ButtonRef, UswdsButtonStyle } from '@/lib/components/uswds/Button';
+import { getCaseNumber } from '@/lib/utils/formatCaseNumber';
+import { OfficeDetails } from '@common/cams/courts';
+import { SuggestedTransferCases, SuggestedTransferCasesImperative } from './SuggestedTransferCases';
+import { FromCaseSummary } from './FromCaseSummary';
+import './PendingTransferOrder.scss';
+
+export type PendingTransferOrderProps = {
+  order: TransferOrder;
+  onOrderUpdate: (alertDetails: AlertDetails, order?: TransferOrder) => void;
+  // TODO: This is a lot of prop drilling. Maybe add a custom hook???
+  officesList: Array<OfficeDetails>;
+};
+
+export type FlexibleTransferOrderAction = Partial<TransferOrderAction> & {
+  newCase?: Partial<CaseSummary>;
+};
+
+export function getOrderTransferFromOrder(order: TransferOrder): FlexibleTransferOrderAction {
+  const { id, caseId } = order;
+  return {
+    id,
+    caseId,
+    orderType: order.orderType,
+  };
```

Comment:
Why did we deconstruct id and caseId but not orderType?  inconsistent.  Also, being such a simple function, is deconstruction really necessary?